docker-compose.yaml

```diff
@@ -7,7 +7,8 @@ services:
       context: .
     environment:
       - MNEMONIC
-      - FORK_URL_BSC
+      - FORK_URL_BSC # Leaving this in here, to potentially not break things?
```

Comment:
@carlomazzaferro can you check this. I renamed this variable, maybe we need to adapt github secrets or other deployment environment variables somewhere?